c2corg_ui/static/menu-discourse/menu.html

```diff
@@ -0,0 +1,117 @@
+## HEAD
+<link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet">
+
+## En-tete
+<aside id="sidemenu">
+  <div class="page-menu">
+    <a href="//www.demov6.camptocamp.org">
```

Comment:
Would it work with the standard ``<base>`` HTML tag?
https://developer.mozilla.org/fr/docs/Web/HTML/Element/base